server/controllers/bookingsController.js

```diff
@@ -1,87 +1,29 @@
 /* eslint-disable dot-notation */
```

Comment:
@Cheza-Dzabala , you should not disable any Eslint Flag. please remove the Eslint comments and ensure you write your codes using the configured Airbnb mode

Reply:
Alright, I will not remove any Eslint flags.